cmd/oras/internal/option/packer.go

```diff
@@ -69,6 +72,29 @@ func (opts *Packer) ExportManifest(ctx context.Context, fetcher content.Fetcher,
 	}
 	return os.WriteFile(opts.ManifestExportPath, manifestBytes, 0666)
 }
+func (opts *Packer) Parse() error {
+	var failedPaths []string
+	currentDir, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	if !opts.PathValidationDisabled && len(opts.FileRefs) != 0 {
```

Comment:
no need to check length of input file references
```suggestion
	if !opts.PathValidationDisabled {
```